src/utils.js

```diff
@@ -3,7 +3,7 @@ export function getBestMessageTranslation(translations, culture, messageKey) {
 
   if (!messageKey || !culture || !translations) return null;
 
-  let lang = translations[_localeFallback(culture)];
+  let lang = translations[getLangFromLocale(culture)];
```

Comment:
Fallback logic is as follows:
if translations has an item for the full locale, use that.
else, try to get a translation for the language only,
else, return null so that the <Text> component renders it's children